### ISSUE TITLE
Report the results of w/o GRPO

### ISSUE THREAD

**Issue by LinB203** [yifan123/flow_grpo#33]
Hi, thanks for your wonderful work. I am wonder that the results of `w/o GRPO`. Because the training dataset you said in paper is `Training prompts are generated using official GenEval scripts`. Therefore, maybe we need to compare main results with `w/o GRPO` to verify the gain of GRPO. But I check the paper and do NOT see any results for `w/o GRPO`, and it only has `w/o KL`. Feel free to correct me If I miss anything. Thanks!

**Comment by ICE-Bruce** [comment]
> Hi Bin,
> 
> We recently ran DPO and SFT baselines on Geneval as well — the results are shown in the image below.
> 
> <img alt="Image" width="2000" height="983" src="https://private-user-images.githubusercontent.com/37578514/463988304-9f0b77f1-ec04-4a59-937c-3592c5170e25.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NjQ1MTE4MTIsIm5iZiI6MTc2NDUxMTUxMiwicGF0aCI6Ii8zNzU3ODUxNC80NjM5ODgzMDQtOWYwYjc3ZjEtZWMwNC00YTU5LTkzN2MtMzU5MmM1MTcwZTI1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTExMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUxMTMwVDE0MDUxMlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNkYWMyZTMyMzM4Mzg1NGI4Zjk2Y2EyZDdkNjQ4ZWMwNGQ5OTkxODBlMTg5MmJkZmM4OWUxYmYwZmQ0MmYxODImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.mfKCScwEmpiQn8RijOMHqtNx1-KNb6ipYkBsurdo7aY">

Hello, I noticed that your training code can not be able to display the results on the official Geneval test set directly? It seems I need to do some additional inference to get the curve with official test set?
